README.md

```diff
@@ -1,23 +1,7 @@
-**This repo is for students to test out their first pull request**
+I have made very meaningful edits to this file.
 
-Students - click below to see the file you'll need to edit:
-- [Spring 2019](https://github.com/joewadcan/FirstPullRequest/tree/master/2019_Spring)
+This will enable me to do a pull request.
 
-**Helpful resources**
+Pull requests are neat. Also, The Internet. 
 
-These videos were included on bCourses, and are a very helpful demo on how GitHub works:
-
-1. GitHub for Poets - Intro (13m video): https://www.youtube.com/watch?v=BCQHnlnPusY 
-
-2. GitHub for Poets - Branching (12m video): https://www.youtube.com/watch?v=oPpnCh7InLY 
-
-3. GitHub for Poets - Forks & Pull Requests (13m video): https://www.youtube.com/watch?v=_NrSWLQsDL4
-
- 
-<details><summary>Previous semesters</summary>
- 
- ### 2018
- - Spring 2018: https://github.com/joewadcan/FirstPullRequest/tree/master/2018_Spring
- - Fall 2018:(https://github.com/joewadcan/FirstPullRequest/tree/master/2018_Fall)
-
-</details>
+The Internet is a band.
```

Comment:
Love the boldness, but sadly I can't accept a change to this file. 😢 These are my directions and I'd like folks to know what they're doing.